workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsTabContent.tsx

```diff
@@ -55,7 +51,7 @@ export const WorkflowDetailsTabContent = ({
             <WorkflowEditor
               kind={EditorViewKind.EXTENDED_DIAGRAM_VIEWER}
               workflowId={workflowId}
-              format={workflowFormat}
+              format={workflowOverviewDTO?.format}
```

Comment:
what if `workflowOverviewDTO` is `undefined`?

Reply:
I think the whole page should not show anything if workflowOverviewFormatDTO is undefined
It should be handled in WorkflowPage

Reply:
@LiorSoffer 
WDYT of WorkflowOverviewDTO property being mandatory
In case it's undefined WorkflowPage should show an error and no content